src/js/utils/mixins.js

```diff
@@ -29,12 +29,23 @@ export const fontSize = (size, lineHeight) => css`
     }px`};
 `;
 
-export const breakpointStyle = (breakpoint, content) => css`
-  @media only screen ${breakpoint.value &&
-    `and (max-width: ${breakpoint.value}px)`} {
-    ${content};
-  }
-`;
+export const breakpointStyle = (breakpoint, content, responsive) => {
+  const st = 
+  responsive === 'container'
+    ? css`
+        @container ${breakpoint.value && `(max-width: ${breakpoint.value}px)`} {
```

Comment:
Will using `@container` cause issues for anyone using v5 of styled-components with Grommet?

Reply:
If I remember correctly it won't create useful css and simply won't work. Either way we'll have to document that this functionality requires styled-components v6+ if you want to use it

Reply:
Ah okay thanks for the explaination